### PR TITLE
Cobra upgrade

### PR DESCRIPTION
**Pull request by BLasan**
## Purpose
Upgrade cobra version and update markdown files.

### CODE REVIEW

**Review by wasuradananjith — APPROVED** [review]

File: import-export-cli/docs/apictl_get_policies_api.md [hunk 1]
```diff
@@ -13,22 +13,21 @@ apictl get policies api [flags]
...
 
 ### Options
 
 ```
-  -e, --environment string   Environment of the API Policies to be fetched
-      --format string        Pretty-print API Policies using Go Templates. Use "jsonArray" to list all fields
-  -h, --help                 help for API Policies
-  -l, --limit string         Limit the number of policies fetched
-  --all                      Fetch all available API Policies
+      --all                  Get all API Policies
```
Comment (sajinieKavindya) [inline import-export-cli/docs/apictl_get_policies_api.md:26]: Formatting issue? unnecessary leading white spaces?

File: import-export-cli/docs/apictl_import_policy.md [hunk 2]
```diff
@@ -13,8 +13,8 @@ apictl import policy [flags]
 ### Examples
 
 ```
-apictl import policy rate-limiting -f ~/CustomPolicy -e production --u
-apictl import policy api -f ~/AddHeader -e production
+apictl import policy rate-limiting -f ~/CustomPolicy -e production -u
+apictl import policy api  -f ~/AddHeader -e production
```
Comment (sajinieKavindya) [inline import-export-cli/docs/apictl_import_policy.md:17]: can we get rid of this extra white space before `-f`?

File: import-export-cli/docs/apictl_get_policies_api.md (hunk 1 above)
Comment (BLasan) [reply import-export-cli/docs/apictl_get_policies_api.md:26]: No, `--all` does not have the shorthand value (i.e  flag with only `-` not `--`). Same as this one https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/docs/apictl_export.md?plain=1#L35

File: import-export-cli/docs/apictl_import_policy.md (hunk 2 above)
Comment (BLasan) [reply import-export-cli/docs/apictl_import_policy.md:17]: fixed via https://github.com/wso2/product-apim-tooling/pull/961/commits/a00e6e265ef72b3e7a9be54d43b92d9d5c5951cb

**Pull request merged**